asb-ballerina/Ballerina.toml

```diff
@@ -1,8 +1,8 @@
 [package]
-distribution = "2201.3.0"
+distribution = "2201.3.1"
 org = "ballerinax"
 name = "asb"
-version = "2.4.0"
+version = "2.5.0"
```

Comment:
Don't we need to release this as a major update `3.0.0`? 

WDYT? @LakshanSS @indikasampath2000 @abeykoon

Reply:
If we introduce any api changes or any major breaking changes, we can go to `3.0.0`.
If not, we can go with `2.5.0`

Reply:
Seems like there are major changes done. So lets go with `3.0.0`